stdlib/_thread.pyi

```diff
@@ -32,7 +33,19 @@ TIMEOUT_MAX: float
 if sys.version_info >= (3, 8):
     def get_native_id() -> int: ...  # only available on some platforms
     @final
-    class _ExceptHookArgs(Tuple[Type[BaseException], Optional[BaseException], Optional[TracebackType], Optional[Thread]]):
+    class _ExceptHookArgs(structseq[Any]):  # Constructor takes an iterable of length 4.
+        @overload  # type: ignore[override]
+        def __getitem__(self, __i: Literal[0]) -> Type[BaseException]: ...
+        @overload
+        def __getitem__(self, __i: Literal[1]) -> BaseException | None: ...
+        @overload
+        def __getitem__(self, __i: Literal[2]) -> TracebackType | None: ...
+        @overload
+        def __getitem__(self, __i: Literal[3]) -> Thread | None: ...
+        @overload
+        def __getitem__(self, __i: SupportsIndex) -> Any: ...
+        @overload
+        def __getitem__(self, __i: slice) -> Sequence[Any]: ...
```

Comment:
This should probably return `Tuple[Any, ...]`. The same goes for other `__getitem__` definitions too.